tests/test_core.py

```diff
@@ -5,7 +5,6 @@
 import networkzero as nw0
 nw0.core._enable_debug_logging()
 
-is_valid_ip = nw0.core.is_valid_ip
```

Comment:
I would've changed this, but you weren't actually using it at all, so it went on annual leave.